server/historian/packages/historian-base/package.json

```diff
@@ -25,9 +25,10 @@
     "@fluidframework/common-utils": "^0.24.0",
     "@fluidframework/gitresources": "^0.1012.0",
     "@fluidframework/protocol-definitions": "^0.1014.0",
-    "@fluidframework/server-services": "^0.1012.2",
-    "@fluidframework/server-services-core": "0.1014.0-7764",
-    "@fluidframework/server-services-utils": "^0.1012.2",
+    "@fluidframework/server-services": "0.1016.1-11989",
+    "@fluidframework/server-services-client": "0.1016.1-11989",
+    "@fluidframework/server-services-core": "0.1016.1-11989",
+    "@fluidframework/server-services-utils": "0.1016.1-11989",
```

Comment:
Note to self: these versions need to be updated with a new minor released version:
1. To pull Zach's new change in throttler library
2. Not ideal to keep absolute version for dependencies

Reply:
Are the new versions not published yet?

Reply:
I think the better syntax to use here would be `"^0.1016.1-0"` so that this package stays up to date with newer changes and fixes.

Reply:
Not yet, I will update the version once we have a new package published (I will also work with Zach on getting it published).

Reply:
@tanviraumi, using `"^0.1016.1-0"` is only installing version `0.1016.1`, which is no longer the latest `0.1016.1` build. I suspect that R11S codebase should have been bumped from `0.1016.1` to `0.1016.2` or `0.1017.0`. Is that correct?

Reply:
You can check the versions here (https://www.npmjs.com/package/@fluidframework/protocol-base) by clicking on the versions tab. Looks like 1016.1 is latest. If you need more changes to published, let me know and I can kick off a new release

Reply:
Right, it looks like 0.1016.1 is the latest, but actually 0.1016.1-11989 is the latest. It appears that 0.1016.1 broke the pre-release convention. In this case, I'm saying that 0.1016.1-11989 should actually be 0.1016.2-11989 or 0.1017.0-11989 because it is no longer a pre-release of 0.1016.1. Perhaps I am misunderstanding though
![image](https://user-images.githubusercontent.com/14362200/103301533-7a51f880-49b6-11eb-9b44-681006af77d1.png)

Reply:
I am not aware of such conventions. We just release them whenever needed and the build number shows up as a suffix.

Reply:
>Precedence refers to how versions are compared to each other when ordered.
> 1. Precedence MUST be calculated by separating the version into major, minor, patch and pre-release identifiers in that order (Build metadata does not figure into precedence).
> 2. Precedence is determined by the first difference when comparing each of these identifiers from left to right as follows: Major, minor, and patch versions are always compared numerically.
> Example: 1.0.0 < 2.0.0 < 2.1.0 < 2.1.1.
> 3. When major, minor, and patch are equal, a pre-release version has lower precedence than a normal version:
> Example: 1.0.0-alpha < 1.0.0.
>
> — [Semver Precedence](https://semver.org/#spec-item-11)

It appears that the process for updating the version in the code is typically done manually (see #4632), but it was not done after releasing 0.1016.1